packages/perseus/src/widgets/passage-ref.jsx

```diff
@@ -65,7 +76,7 @@ class PassageRef extends React.Component<Props, State> {
     }
 
     getUserInput: () => UserInput = () => {
-        return WidgetJsonifyDeprecated.getUserInput.call(this);
+        return removeDenylistProps(this.props);
```

Comment:
so this new function name no longer indicates that the operation is suboptimal. From the widget-jsonify-deprecated:
```
/* Free implementation of getUserInput. This should be used sparingly, since it
 * just returns all the widget's props rather than picking out those which were
 * input by the user.
 */
```
Can you add a `TODO` or something similar indicating that this `getUserInput` really should be tuned to pick out only those props that were input by the user?

Reply:
Thanks for pointing this out! Just added a TODO. Let me know if you think it needs to be updated/changed at all.